### DISCUSSION TITLE
Remove audio / sounds from gba rom totally

### DISCUSSION THREAD

**Discussion by tattoo-girl** [haven1433/HexManiacAdvance#107] [Q&A] [answered]
This program seems to be what I've been looking for.

I wonder if it can delete / remove audio or sounds totally from a gba rom ?

I want to completely remove audio or sounds.  I mean total silence from start to finish.

Can i achieve this?

If so, then please 🙏  a little step by step guide on how I can do it.

Thank you!

**Comment by haven1433** [comment, marked as answer]
HMA doesn't mess with audio at all. I think most people use a tool called Sappy to work with sounds / music.

**Reply by tattoo-girl** [reply]
I've tried sappy.  Only the version in French seemed to be working.
And then, I don't understand how to use it cos it's all in French. So I don't know how to get it to delete all audio & sounds. 

If you have experience with sappy, I would appreciate some tips or guidance.

HMA on the other hand is real easy to navigate. You have a good thing going here. 👍

any suggestion or advice on what I can do to solve my problem? That is to totally eliminate sounds & audio from a gba rom?

**Reply by haven1433** [reply]
I'm glad you like the tool :) but unfortunately I have no experience working with sappy or GBA sound in general. I'm not sure how to help you, but there's a lot of experts in the following rom hacking discord servers that may have some relevant experience:

[Pokecommunity](https://discord.gg/pokecommunity)

[Anthroyd's Server](https://discord.gg/ZEATnPKa)

There's also the pokecommunity forums and pokecommunity reddit, though I rarely find answers I'm looking for there.

Sorry I couldn't be more help! I've not dabbled in sound data at all. I probably should at some point :D

**Reply by tattoo-girl** [reply]
![IMG_20230122_093200_927](https://user-images.githubusercontent.com/123066395/213907416-7ece9945-9749-40e8-b0ce-2582a513573a.jpg)

Yeah ok, that would be nice too, cos control of sound data may make HMA an even more of a complete tool. 

Sorry for the mistake, I got confused because I saw " sound " on HMA as in the image I've shared , so I thought it could be used to edit rom sounds.

I would check some of the forums you've suggested.

Thank You again for taking the time to respond, I really appreciate.
